src/quo2/components/settings/category/view.cljs

```diff
@@ -0,0 +1,19 @@
+(ns quo2.components.settings.category.view
+  (:require
+    [quo2.core :as quo]
```

Comment:
oh I think this leads to circular dependencies, better to just import directly from the file within quo2